cli.js

```diff
@@ -56,10 +74,44 @@ function run(file) {
 			return;
 		}
 
```

Comment:
How can it both return a promise and emit events? I had this in mind:

```js
function fork (file) {
  return new Promise(function (resolve, reject) {
    var ps = child_process.fork(file);

    // need also to emit test events somehow

    ps.on('close', function (code) {
      if (code > 0) return reject();

      resolve();
    });
  });
}
```

As a workaround, we could accept test listener:

```js
function fork (file, test) {
  ...
  ps.on('data', test);
  ...
}
```